glitter/blocks/video/models.py

```diff
@@ -37,7 +38,8 @@ def get_embed_url(self):
     def save(self, force_insert=False, force_update=False, using=None, update_fields=None):
         """ Set html field with correct iframe. """
         if self.url:
-            self.html = """<iframe src="{}" frameborder="0" allowfullscreen></iframe>""".format(
-                self.get_embed_url()
+            self.html = """<iframe src="{}" frameborder="0" title="{}" allowfullscreen></iframe>""".format(
+                self.get_embed_url(),
+                self.title
```

Comment:
Add the comma at the end! :+1: https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas